### ISSUE TITLE
clang-format.style in Windows?

### ISSUE THREAD

**Issue by flynnwt** [xaverh/vscode-clang-format-provider#6]
On Windows, clang-format is ignoring the -style setting because of invalid value, and using the fallback.  doFormatDocument() is using execFile() to execute the command, which I think is causing the double quotes to be stripped.  I am using this in my preferences (which works on the command line).
```js
"clang-format.style": "{BasedOnStyle: llvm, AlignTrailingComments: true, ColumnLimit: 0}"
```
Is this the correct way to specify it?

**Comment by owt5008137** [comment]
@flynnwt No, you should use standard json but not javascript code.
```
"clang-format.style": '{"BasedOnStyle": "llvm", "AlignTrailingComments": true, "ColumnLimit": 0}'
```

**Comment by xaverh** [comment]
Problem was fixed in Pull Request #8 and pushed as v 0.6.0 to the Visual Studio Code Marketplace.
